checkout.css

```diff
@@ -0,0 +1,186 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap");
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css";
+
+:root {
+	font-size: 62.5%;
+	--primary-color: #ff3f6c;
+	--dark-primary-color: #e63961;
+	--complimentary-text-color: white;
+	--primary-text-color: #3e4152;
+	--secondary-text-color: #696b79;
+	--border-color: #e6e6e6;
+}
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: "Roboto", sans-serif;
+}
+.container {
+	padding: 3rem 0;
+	text-align: center;
+}
+.checkout-heading {
+	display: inline-block;
+	font-size: 2rem;
+	font-weight: 700;
+	color: #282c3f;
+	padding: 3rem 0;
+}
+.checkout-component {
+	width: 65%;
+	display: flex;
+	margin: 0 auto;
+    height: fit-content;
+}
+.card-items {
+	display: flex;
+	flex-direction: column;
+}
+.card-item {
+	display: flex;
+	width: 100%;
+	border: 2px solid var(--border-color);
+	border-radius: 0.35rem;
+	margin: 1rem 0;
+}
+.image-container {
+	width: 157px;
+	height: 148px;
+}
+.image-res {
+	width: 100%;
+	height: auto;
+}
+.card-description {
+	display: flex;
+	flex-direction: column;
+	width: calc(100% - 130px);
+	padding: 1rem;
+	align-items: start;
+}
+.card-item-heading {
+	font-size: 1.6rem;
+	font-weight: 500;
+	color: #282c3f;
+	margin-bottom: 0.3rem;
+}
+.card-item-desc {
+	font-size: 1.2rem;
+	margin-bottom: 0.3rem;
```

Comment:
use px

Reply:
Actually i'm following same format other files. I will be change later.